visualdl/storage/binary_record.h

```diff
@@ -52,42 +51,59 @@ struct BinaryRecord {
     file.write(data_.data(), data_.size());
   }
 
-  const std::string& hash() { return hash_; }
+  static std::string generateFileName() {
```

Comment:
GenerateFileName

according to https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Will do